testsuite/Makefile

```
@@ -93,6 +93,17 @@ OCAMLTESTFLAGS := \
   $(OCAMLTEST_PROMOTE_FLAG) \
   $(OCAMLTEST_KEEP_TEST_DIR_ON_SUCCESS_FLAG)
 
+ifneq ($(USE_RUNTIME),)
+# Check USE_RUNTIME value
+ifeq ($(findstring $(USE_RUNTIME),d i),)
+$(error If set, USE_RUNTIME must be equal to "d" (debug runtime) \
+        or "i" (instrumented runtime))
```

```suggestion
ifeq ($(wildcard $(ROOTDIR)/runtime/ocamlrun$(USE_RUNTIME),)
$(error USE_RUNTIME=$(USE_RUNTIME) but \
        $(ROOTDIR)/runtime/ocamlrun$(USE_RUNTIME) not found.)
```
rather than hard-coding the list? It also guards against trying to run with `USE_RUNTIME=d` and `--disable-debug-runtime` (I realise it's a change from what was there before)